test/functional/apps/discover/_discover.ts

```diff
@@ -19,16 +19,18 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
   const queryBar = getService('queryBar');
   const inspector = getService('inspector');
   const elasticChart = getService('elasticChart');
+  const savedObjects = getService('savedObjects');
+  const supertest = getService('supertest');
   const PageObjects = getPageObjects(['common', 'discover', 'header', 'timePicker']);
+
   const defaultSettings = {
     defaultIndex: 'logstash-*',
   };
 
   describe('discover test', function describeIndexTests() {
     before(async function () {
       log.debug('load kibana index with default index pattern');
-      await esArchiver.load('discover');
-
+      await savedObjects.import('discover')(log)(supertest);
```

Comment:
I think you might still need `await esArchiver.load('empty_kibana');` here to clear out any previous saved objects before adding these saved objects.

Reply:
Forthcoming!